Installation/cmake/modules/UseLASLIB.cmake

```diff
@@ -1,4 +1,4 @@
 # This module setups the compiler for using LASlib library.
 # It assumes that find_package(LASLIB) was already called.
 
-add_definitions(-DCGAL_LINKED_WITH_LASLIB)
+add_definitions(-DCGAL_LINKED_WITH_LASLIB -DUSE_AS_DLL)
```

Comment:
If the CMake command was a `target_compile_definitions`, just for a specific target, that would be fine. But here the line `add_definitions(-DCGAL_LINKED_WITH_LASLIB -DUSE_AS_DLL)` will add `-DUSE_AS_DLL` to all compilation commands of the CMake project.

If that is not too complicated to hide that macro in just one header of CGAL, I would prefer.